packages/fxa-settings/src/pages/ConfirmSignin/index.tsx

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { usePageViewEvent } from '../../lib/metrics';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { RouteComponentProps } from '@reach/router';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+
+export type ConfirmSigninProps = {
+  canGoBack?: boolean;
+  email: string;
+  goBackCallback?: Function;
+  isOpenWebmailButtonVisible: boolean; // TO-DO: Replace broker functionality which gives us this value (provider?)
+};
+
+export type WebmailValues = {
+  buttonText: string;
+  link: string;
+};
+
+const ConfirmSignin = ({
+  canGoBack,
+  email,
+  goBackCallback,
+  isOpenWebmailButtonVisible,
+}: RouteComponentProps & ConfirmSigninProps) => {
+  usePageViewEvent('confirm-signin', {
+    entrypoint_variation: 'react',
+  });
+
+  // TO-DO: Replace utility that gets these values by matching the email provider via regex.
+  const getWebmailValues = (email: string) => {
+    return {
+      buttonText: 'Open Gmail',
+      link: `https://mail.google.com/mail/u/?authuser=${email}`,
+    };
+  };
+
+  const webmailValues = getWebmailValues(email);
+
+  const resendEmail = () => {
+    // TO-DO: resend email to user.
+  };
+
+  /*
+    TO-DO:
+        - Fix up the alert bar for any errors (such as from resending the email) or success.
+        - wire up the functionality to resend an email
+  */
+
+  return (
+    <>
+      <div className="mb-4">
+        <FtlMsg id="confirm-signin-header">
+          <h1 className="card-header">Confirm this sign-in</h1>
+        </FtlMsg>
+      </div>
+      <section>
+        <div className="flex justify-center">
+          <Mail role="img" />
```

Comment:
In her review for SigninTotpCode, [Lauren recommended that we add `aria-label`](https://github.com/mozilla/fxa/pull/14744#discussion_r1069645552) even if these images could be considered decorative.